src/porepy/viz/exporter.py

```diff
@@ -302,6 +302,8 @@ def _write_vtk(self, data, name, g_VTK):
 
         if data is not None:
             for name_field, values_field in data.items():
+                if values_field is None:
+                    continue
                 dataVTK = ns.numpy_to_vtk(values_field.ravel(order='F'),
```

Comment:
Do you see any problem by not write a variable to all dimensions?

One problem I see is that you now will not get an error message if you try to export a non-existing variable